src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -0,0 +1,253 @@
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Mixins, Emit } from 'vue-property-decorator'
+
+// Components
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+// Dialogs
+import { ConfirmRemoveDialog } from '@/components/dialogs'
+
+// Mixins
+import CommonMixin from '@/mixins/common-mixin'
+
+// Interfaces
+import { OrgPersonIF } from '@/interfaces'
+
+@Component({
+  components: {
+    BaseAddress,
+    ConfirmRemoveDialog
+  }
+})
+export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
+  // Store Properties
+  @Prop({ default: '' })
+  private personList: Array<OrgPersonIF>
+
+  @Prop({ default: false })
+  private showErrorSummary: boolean
+
+  @Prop({ default: false })
+  private isSummary: boolean
+
+  // Local Properties
+  readonly tableHeaders: Array<string> = ['Name', 'Mailing Address', 'Delivery Address', 'Roles']
+  private dialog: boolean = false
+  private personId: number
+
+  /**
+   * Determine if Corporation/Firm or Person.
+   * @param filing The filing body which contains the name/title.
+   * @returns The appropriate Corporation or Person name.
+   */
+  private formatName (filing: any): string {
+    return filing.orgName ? filing.orgName
+      : `${filing.firstName} ${filing.middleName || ''} ${filing.lastName}`
+  }
+
+  /**
+   * Identify and confirm the removal of a person/org through dialog.
+   * @param index The active index which is subject to removal.
+   */
+  confirmRemove (index: number): void {
+    this.personId = index
+    this.dialog = true
+  }
+
+  /**
+   * Emit an index and event to the parent to handle removal.
+   * @param index The active index which is subject to removal.
+   */
+  @Emit('removePerson')
+  private emitRemovePerson (index: number): void {
+    this.dialog = false
+  }
+
+  /**
+   * Emit an index and event to the parent to handle editing.
+   * @param index The active index which is subject to change.
+   */
+  @Emit('editPerson')
+  private emitPersonInfo (index: number): void {}
+}
+</script>
+
+<style lang="scss" scoped>
+#people-roles {
+  margin-top: 1rem;
+}
+
+.people-roles-summary-header {
+  display: flex;
+  background-color: rgba(1, 51, 102, 0.15);
+  padding: 1.25rem;
+}
+
+.people-roles-invalid-message {
+  padding: 1.25rem;
+  font-weight: bold;
+  color: #1976d2;
+}
+
+.people-roles-header {
+  padding: .5rem 1.25rem .5rem 1.25rem;
+  font-size: 12px;
+  margin-top: 1rem;
+}
+
+.people-roles-content {
+  margin-top: .5rem;
+  padding: .5rem 1.25rem .5rem 1.25rem;
+  border-top: 1px solid #d1d1d1;
+  font-size: 14px;
+
+  .people-roles-title {
+    color: #474747;
+  }
+
+  .actions {
+    position: absolute;
+    right: 0;
+
+    .edit-action {
+      border-right: 1px solid #d1d1d1;
+    }
+
+    .v-btn {
+      min-width: 1rem;
+    }
+
+    .v-btn + .v-btn {
+      margin-left: 0.5rem;
+    }
+  }
+}
+
+.col {
+  padding: .25rem;
+
+  .col-roles {
+    padding: 0rem!important;
```

Comment:
Missing space?

Reply:
Wanted padding for the column but didn't want that inherited ( double padded ) for the child column

Reply:
Don't we need a space between `rem` and `!important`?

Reply:
ohh. The linters say no, my sanity says yes. But, i just removed the `rem` anyways because thats not required with a falsy value.

Reply:
I don't think it's interpreted as falsy. I think 0px = 0em = 0rem = just 0. There's probably actual guidance on this somewhere but I'm going on experience here :)

Reply:
Interesting, i trust you! Removed the rem either way 👍